### DISCUSSION TITLE
TeamTalk v. 5.8.2.5053 - Two Bugs That I Found

### DISCUSSION THREAD

**Discussion by StormProductionsMusic** [BearWare/TeamTalk5#1078] [General]
Hi,

I found two weird bugs in the new beta TeamTalk in regarding to its usage with NVDA:

1. When I'm focused on an user and that user starts streaming a file, TeamTalk starts spamming the screen reader with a lot of Streaming Media File.
2. When I'm trying to move 2 or more users from a channel to another, I select them as I normally would with ctrl+alt+x, then ctrl+alt+v. But looks like only one of the users I select is getting moved, the rest stay on the channel I tried to move from.

Hope they'll get fixed in the next update.

**Comment by CoBC** [comment]
Hi,
for the first point, I can confirm this, if the focus is on an user in tree, and this user change status, screenreader is a bit upset. I don't have solution for now.
For the second point, I guess you use Qt client, so you don't have to press CTRL-Alt-X on each user you want to move. In Qt client the selection works like in all other Windows applications, you have to select all users you want to move using Shift-Arrow or CTRL-Arrow/CTRL-Space, and press CTRL-Alt-X only one time.

**Reply by bear101** [reply]
Do you know which text string is being spammed?

Events CLIENTEVENT_CMD_USER_UPDATE and CLIENTEVENT_USER_STATECHANGE should only be triggered on changes. I was suspecting CLIENTEVENT_STREAM_MEDIAFILE but it's only during local media streaming and not from a remote user.
